components/blitz/resources/omero/cmd/Mail.ice

```diff
@@ -28,10 +28,10 @@ module omero {
          * and/or specific users.
          * 
          * examples:
-         *  - omero.cmd.SendEmailRequest(subject, body)
+         *  - omero.cmd.SendEmailRequest(subject, body, everyone)
          *		sends message to everyone who has email set
          *		and is an active user
-         *  - omero.cmd.SendEmailRequest(subject, body, inactive)
+         *  - omero.cmd.SendEmailRequest(subject, body, everyone, inactive)
```

Comment:
Here `inactive=` would also need a keyword in addition to `everyone=` as above (https://github.com/openmicroscopy/openmicroscopy/pull/3434/files#r24334676)